.evergreen/atlas/teardown.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+# Handle proper teardown of Atlas.
```

Comment:
imo this documentation comment is unnecessary: the name of the script and its parent directory clearly describes the purpose of this script. Suggest its removal from this + similar scripts.

Reply:
Done